blogposts/2021/release-3.30.md

```diff
@@ -0,0 +1,204 @@
+    url:
+    category:
+  - description: "Go blank identifier assignments `var _ = ...` are no longer incorrectly included."
+    url:
+    category:
+  - description: "Go symbols defined within functions, e.g. a `var` inside a `func` scope are no longer incorrectly included."
+    url:
+    category:
+  - description: "`Functions`, `Variables`, and other top-level sections are no longer rendered empty if there are none in that section."
+    url:
+    category:
+  - description: A new test suite for LSIF indexers implementing the Sourcegraph documentation extension to LSIF is available.
+    url: "https://github.com/sourcegraph/lsif-static-doc"
+    category:
+  - description: "We now emit the LSIF data needed to in the future support \"Jump to API docs\" from code views, \"View code\" from API docs, usage examples in API docs, and search indexing."
+    url:
+    category: Search
+  - description: Various UI style issues, color contrast issues, etc. have been fixed.
+    url:
+    category:
+  - description: Major improvements to the GraphQL APIs for API documentation.
+    url:
+    category: API
+---
+
+Sourcegraph 3.30 is now available! For this release, we introduced
+
+## Update
+
+### Batch Changes: Publish and unpublish changests from the GUI
+
+Batch Changes features a powerful idempotent [spec-based workflow](https://docs.sourcegraph.com/batch_changes/explanations/batch_changes_design), where the spec file defines the batch change. In particular, the [`published`](https://docs.sourcegraph.com/batch_changes/references/batch_spec_yaml_reference#changesettemplate-published) property defines if changesets are published to the codehost or not.
+
+This workflow is great for reproducibility and having a completely CLI-based workflow, and we are keeping it!
+
+_In addition_, we are allowing changesets to be published and unpublished directly from the GUI, to make the workflow more approachable to new users. Simply omit the `published` field in the specs, and the GUI will become the source of truth for publishing changesets.
+
+<img src="https://storage.googleapis.com/sourcegraph-assets/blog/3.30/publish-from-gui.gif" alt="Publish changesets from GUI demo" />
+
+### New code search reference
+
+* We've added a new search reference to the search results sidebar. The search reference lists search filters which can be expanded to display documentation and examples. The reference can be used to learn about filters or as a shortcuts to using the fitlers in queries. 
+
+<!--TODO a small GIF a user clicking a repo: filter and using autocomplete to resolve github.com\/sourcegraph/sourcgraph$ -->
+
+### Code Insights Dashboards 
+
+<!--TODO a small GIF of adding code insights to a dashboard and viewing that dashboard -->
```

Comment:
@Joelkw - please add the GIF.

Reply:
Yep! Sorry, left you a comment last night on this issue here rather than this specific line, but aware it's still missing: https://github.com/sourcegraph/about/pull/3731#issuecomment-883813883

Reply:
```suggestion
<p><video autoplay loop muted playsinline style="width:600px">
  <source src="https://sourcegraphstatic.com/blog/3.30/insights_dashboards.mp4" type="video/mp4">
 </video></p>
```